circle.yml

```diff
@@ -31,10 +27,12 @@ dependencies:
     - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     - sudo dpkg -i google-chrome.deb
     - sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
-    - pip install -U pip
-    - pip install -r tests/testrunner/requirements.txt
-    - python setup.py develop:
-        pwd: tests/testrunner
+    - |
+      login="$(aws ecr get-login)"
```

Comment:
You __should__ be able to just `aws login...`

Reply:
we spent a while fudging with this and it just wouldn't play nice, and this sequence is known to work. maybe a small ticket for a rainy day, as we use it across all the sdks.

Reply:
ack